swagger.yaml

```diff
@@ -0,0 +1,515 @@
+swagger: "2.0"
+info:
+  version: "v3"
+  title: "Billing Account API"
+# during production,should point to your server machine
+host: localhost:3000
+basePath: "/v3"
+# during production, should use https
+schemes:
+- "http"
+produces:
+- application/json
+consumes:
+- application/json
+
+securityDefinitions:
+  Bearer:
+    type: apiKey
+    name: Authorization
+    in: header
+
+paths:
+  /billingaccounts:
```

Comment:
Yep it is, just makes it a little difficult to read especially since the naming standards for fields is camelCase.